scalajvm/test/ApproverModelSpec.scala

```diff
@@ -11,9 +12,11 @@ class ApproverModelSpec extends Specification {
     ("ForNeVeR", "fornever@loglist.xyz")
   )
 
+  val fakeApplication = GuiceApplicationBuilder().build()
+
   "The Approvers model" should {
     "be able to add new approvers and return them back" in {
-      running(FakeApplication()) {
+      running(fakeApplication) {
```

Comment:
Warning: this isn't tested yet. It compiles, but I never ran it because it requires a production-like environment. Should I write a separate docker-compose.yamll for that?

Reply:
Will test when setting up the CI.